sdk/cdn/arm-cdn/src/models/index.ts

```diff
@@ -1173,14 +1192,6 @@ export interface UrlSigningParamIdentifier {
  * Defines the parameters for the Url Signing action.
  */
 export interface UrlSigningActionParameters {
```

Comment:
Is this used as input for any of the user facing methods? If yes, then isnt it a breaking change that members of this interface are being removed?

Reply:
@ramya-rao-a 
I found that the definition of ```subscriptionId``` in [cdnwebapplicationfirewall.json](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/cdnwebapplicationfirewall.json#L891) and [afdx.json](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/afdx.json#L5547),[cdn.json](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/cdn.json#L5622) exists problem of Inconsistent word case,and ```apiversion``` also has same problem.thanks